packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -203,8 +204,11 @@ Host platform code is in the android/ and ios/ directories under $relativePlugin
       'androidIdentifier': _createAndroidIdentifier(projectName),
       'iosIdentifier': _createUTIIdentifier(projectName),
       'description': projectDescription,
-      'flutterPackagesDirectory': flutterPackagesDirectory,
+      'dartSdk': '$flutterRoot/bin/cache/dart-sdk',
+      'flutterPackagesDirectory': '$flutterRoot/packages',
```

Comment:
When calculating `flutterPackagesDirectory` above, we use`fs.path.join()` - we should do the same here as well.